common/config_parser.hpp

```diff
@@ -89,9 +89,11 @@ class MenderConfigFromFile {
 	/** Connectivity connection handling and transfer parameters */
 	ClientConnectivity connectivity;
 
-	/** Rootfs device paths */
-	string rootfs_part_A;
-	string rootfs_part_B;
+	/** Rootfs device paths. These are not parsed by the client anymore, since rootfs updates
+		are now handled by an update module. But for historical reasons, they still share config
+		files, so these options can still be in the file. */
+	// string rootfs_part_A;
```

Comment:
Q: Are we, if they are not set, just going to ignore it until the UM fails?

Reply:
Yep! It's out of our area of responsibility, basically.